docs/en/04_Changelogs/3.2.0.md

```diff
@@ -1,56 +1,185 @@
- * `Cookie::forceExpiry()` was removed. Use `Cookie::force_expiry()` instead
- * `Object` statics removal: `get_static()`, `set_static()`, `uninherited_static()`, `combined_static()`,
+* `Cookie::forceExpiry()` was removed. Use `Cookie::force_expiry()` instead
+* `Object` statics removal: `get_static()`, `set_static()`, `uninherited_static()`, `combined_static()`,
  `addStaticVars()` and `add_static_var()` removed. Use the Config methods instead.
- * `GD` methods removed: `setGD()`, `getGD()`, `hasGD()`. Use `setImageResource()`, `getImageResource()`, and `hasImageResource()` instead
- * `DataExtension::get_extra_config()` removed, no longer supports `extraStatics` or `extraDBFields`. Define your
+* `GD` methods removed: `setGD()`, `getGD()`, `hasGD()`. Use `setImageResource()`, `getImageResource()`, and `hasImageResource()` instead
+* `DataExtension::get_extra_config()` removed, no longer supports `extraStatics` or `extraDBFields`. Define your
  statics on the class directly.
- * `DataList::getRange()` removed. Use `limit()` instead.
- * `SQLMap` removed. Call `map()` on a `DataList` or use `SS_Map` directly instead.
- * `Profiler` removed. Use xhprof or xdebug for profiling instead.
- * `Aggregate` removed. Call aggregate methods on a `DataList` instead e.g. `Member::get()->max('LastEdited')`
- * `MySQLDatabase::set_connection_charset()` removed. Use `MySQLDatabase.connection_charset` config setting instead
- * `SQLConditionalExpression/SQLQuery` `select()`, `limit()`, `orderby()`, `groupby()`, `having()`, `from()`, `leftjoin()`, `innerjoin()`, `where()` and `whereAny()` removed.
+* `DataList::getRange()` removed. Use `limit()` instead.
+* `SQLMap` removed. Call `map()` on a `DataList` or use `SS_Map` directly instead.
+* `Profiler` removed. Use xhprof or xdebug for profiling instead.
+* `Aggregate` removed. Call aggregate methods on a `DataList` instead e.g. `Member::get()->max('LastEdited')`
+* `MySQLDatabase::set_connection_charset()` removed. Use `MySQLDatabase.connection_charset` config setting instead
+* `SQLConditionalExpression/SQLQuery` `select()`, `limit()`, `orderby()`, `groupby()`, `having()`, `from()`, `leftjoin()`, `innerjoin()`, `where()` and `whereAny()` removed.
  Use `set*()` and `add*()` methods instead.
- * Template `<% control $MyList %>` syntax removed. Use `<% loop $MyList %>` instead.
- * Object::singleton() method for better type-friendly singleton generation
-
-### CMS
-
- * `SearchForm::getSearchQuery` no longer pre-escapes search keywords and must be cast in your template
+* Template `<% control $MyList %>` syntax removed. Use `<% loop $MyList %>` instead.
+* Object::singleton() method for better type-friendly singleton generation
+* Removed `Member.LastVisited` and `Member.NumVisits` properties, see
+  [Howto: Track Member Logins](doc.silverstripe.org/framework/en/trunk/howto/track-member-logins) to restore functionality as custom code
```

Comment:
Oh, and that tracking member logins doesn't actually give correct code for 3.2 as it's using `DB::query`...